REFERENCE.md

```diff
@@ -1,19 +1,18 @@
 # Reference
-
 <!-- DO NOT EDIT: This document was generated by Puppet Strings -->
 
 ## Table of Contents
 
-### Classes
+**Classes**
 
-#### Public Classes
+_Public Classes_
```

Comment:
Could it be that you have an old version of strings installed and that it's using an old template?

Reply:
You are right, I have updated this pull request with the latest version.